internal/minikube/docker.go

```diff
@@ -0,0 +1,42 @@
+package k8s
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+)
+
+// This isn't perfect (because it won't unquote the value right) but
+// it's good enough for 99% of cases.
+var envMatcher = regexp.MustCompile(`export (\w+)="([^"]+)"`)
+
+func DockerEnv() (map[string]string, error) {
```

Comment:
do we care about running this with a ctx/timeout? this call doesn't seem particularly risky, but idk, good practice? 🤷‍♀️

Reply:
good point, added!